### PR TITLE
EBODAC-801 "QA - Booking Application - (Screen 4) - Reschedule a visi…

### PR DESCRIPTION
**Pull request by tomaszmueller**
…t Test"

### CODE REVIEW

**Comment by tomaszmueller** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppRescheduleVisitPage.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                return 10;
+            case "december":
+                return 11;
+            default:
+                return -1;
+        }
+
+    }
+}
+
```
Comment (kkaczmarczyk) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppRescheduleVisitPage.java]: redundant new lines

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/BookingApplicationRescheduleVisitUiTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package org.motechproject.ebodac.uitest.test;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.motech.page.LoginPage;
+import org.motech.test.TestBase;
+import org.motechproject.ebodac.uitest.page.*;
```
Comment (kkaczmarczyk) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/BookingApplicationRescheduleVisitUiTest.java]: Try not to use wildcard import, they are bad for maintainability of code. You can force intellij not to do it
http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Checkstyle rule about this:
http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppRescheduleVisitPage.java [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+package org.motechproject.ebodac.uitest.page;
+
+import org.motech.page.AbstractBasePage;
+import org.openqa.selenium.*;
```
Comment (kkaczmarczyk) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppRescheduleVisitPage.java]: Try not to use wildcard import, they are bad for maintainability of code. You can force intellij not to do it
http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Checkstyle rule about this:
http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport

**Comment by tomaszmueller** [comment]
@kkaczmarczyk, @pmuchowski fixed and updated

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by kkaczmarczyk** [comment]
test this please

**Pull request merged**